packages/tokens/styles/layout.scss

```diff
@@ -0,0 +1,5 @@
+@import "type";
+
+$ca-layout-max-width: 1416px;
+$ca-layout-side-padding: 24px;
+$ca-layout-content-width: 1080px;
```

Comment:
Not 100% sure how I feel about these ones. They're very layout focused.

Reply:
I guess layouts and content widths are design decisions, so it makes sense for them to be tokenised, even if they can be considered legacy tokens (we may have a completely different non backwards compatible approach tokens when it comes time to doing layout for the design vision)

Reply:
Also, we have a `Layout` component. There's an ongoing problem with the question of where to put these values – if we put them with the Layout component, people in other repos then use those styles directly and it becomes brittle, plus a few other Kaizen components will end up doing the same :( I'm pretty sure the reason these are here now is that they used in be in the Layout component and got moved out once other components like `TitleBlock` and `EmptyState` started needing them.

"Layout tokens" are something we should make into a wider conversation I reckon.

See this convo @dougmacknz and I having btw: https://github.com/cultureamp/kaizen-design-system/pull/103#issuecomment-548649459